pkgs/development/tools/iat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "iat";
+  version = "0.1.7";
+
+  src = fetchurl {
+    url = "https://cfhcable.dl.sourceforge.net/project/iat.berlios/iat-0.1.7.tar.gz";
```

Comment:
Try and find a `mirror://sourceforge/` URL that would work here. There should be examples of doing this nearby.

Reply:
Nice catch, done.